lib/profiles.js

```diff
@@ -15,12 +15,13 @@ const KMS_MODULE = 'ssm-v1';
  * Creates a Profile for a given account
  *
  *
- * @param {String} accountId the id of the account to associate with the
- *                           Profile.
+ * @param {string} accountId - Id of the account to associate with the Profile.
+ * @param {object} [didMethodOptions] - Hashmap of optional DID method options.
+ * @param {string} [didMethodOptions.method] - Supported: 'key' and 'v1'.
  *
  * @return {Promise<Profile>} resolves to a profile's settings.
  */
-exports.create = async ({accountId} = {}) => {
+module.exports.create = async ({accountId, didMethodOptions} = {}) => {
```

Comment:
I was thinking more like this:

```suggestion
module.exports.create = async ({accountId, didMethod, didMethodOptions = {}} = {}) => {
```

Does anyone else have an opinion? @mattcollier, @gannan08 ?

Reply:
@dlongley Ah, sure thing. I'll split it out.

Reply:
@dlongley renamed.